android/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id("com.android.application")
+    kotlin("android")
+}
+
+android {
+    namespace = "vn.luongvo.kmm.survey.android"
+    compileSdk = 32
+    defaultConfig {
+        applicationId = "vn.luongvo.kmm.survey.android"
+        minSdk = 24
+        targetSdk = 32
+        versionCode = 1
+        versionName = "1.0"
+    }
+    buildFeatures {
+        compose = true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.3.0"
+    }
+    packagingOptions {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
+        }
+    }
+    buildTypes {
+        getByName("release") {
+            isMinifyEnabled = false
+        }
+    }
+}
+
+dependencies {
+    implementation(project(":shared"))
+    implementation("androidx.compose.ui:ui:1.2.1")
+    implementation("androidx.compose.ui:ui-tooling:1.2.1")
+    implementation("androidx.compose.ui:ui-tooling-preview:1.2.1")
+    implementation("androidx.compose.foundation:foundation:1.2.1")
+    implementation("androidx.compose.material:material:1.2.1")
+    implementation("androidx.activity:activity-compose:1.5.1")
+}
```

Comment:
Is not including last break line normal? Do you normally turn on this setting?
<img width="434" alt="Screen Shot 2022-11-22 at 11 08 07" src="https://user-images.githubusercontent.com/6356137/203219056-70e097c3-0012-46ad-abb2-7db7805c45c0.png">